utils/python/CIME/XML/namelist_defaults.py

```diff
@@ -0,0 +1,113 @@
+"""Interface to `namelist_defaults.xml`.
```

Comment:
Okay I see where you are coming from but it seems like a lot of python code is being written to support this format that we plan to get rid of and that a lot of reusable code in entryid is not being taken advantage of.